applications/feedback_sentiment_analyzer/README.md

```diff
@@ -0,0 +1,42 @@
+# Feedback Sentiment Analyzer (FSA)
+
+Feedback Sentiment Analyzer (FSA) is an example application that showcases AWS services and SDKs.
+
+Specifically, it solves a ficticious use case of a hotel in Paris, France, which receives feedback from guests in a variety of foreign languages.
```

Comment:
We discussed the readme being the place where we give a full explanation of what the app does. I think that's missing here. Here's what I put in the ruby xml block.

```
    This example application analyzes and stores customer feedback cards. Specifically,
    it fulfils the need of a fictitious hotel in Paris, France. The hotel recieves feedback
    from guests in various languages in the form of physical comment cards. That feedback
    is uploaded into the app through a web client.
```